Resources/Prototypes/_Sunrise/Entities/Clothing/Hands/gloves.yml

```
@@ -23,7 +23,7 @@
     - type: Insulated
 
 - type: entity
-  parent: ClothingHandsBase
+  parent: [ ClothingHandsGlovesCombat, BaseSecurityContraband ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)